app/app/Http/Controllers/PasswordRecoveryController.php

```diff
@@ -36,6 +36,21 @@ public function sendEmail(Request $request)
             return view('pages.password_recovery.unmatched_email');
         }
 
+        if( $matching_user->email_verification_time == null) {
+            $confirmationLink = BaseUser::generateConfirmationLink($matching_user);
+            Mail::send(new ConfirmationMail(
```

Comment:
Where is ConfirmationMail defined?  I guess you want it to be the one from App\Mail\ConfirmationMail but since that's not mentioned in the use statements, it is trying to find ConfirmationMail in App\Controllers namespace and failing.

![image](https://user-images.githubusercontent.com/11838191/44203649-bf469e80-a11d-11e8-928a-06e4b6cc11fe.png)

Reply:
I created and merged pull request #682 to fix this.